### PR TITLE
Security scanner configuration - first step

### PR DESCRIPTION
**Pull request by jaroslaw-pieszka**
**Description**

Configuration for the new approach (self-service) of security scanners configuration

Changes proposed in this pull request:

- dynamic configuration while creating module template to allow post-release scanning in respective channels (regular or fast)

**Related issue(s)**

### CODE REVIEW

File: scripts/create_scan_config.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being maskedPORT=5001
+
+IMAGE=${1}
+echo "Creating security scan configuration file:"
+cat <<EOF | tee ${FILENAME}
```
Comment (PK85) [inline scripts/create_scan_config.sh:18]: whats about labels like team gopher?

File: scripts/release.sh [hunk 2]
```diff
@@ -29,9 +29,16 @@ uploadFile() {
   fi
 }
 
-echo "PULL_BASE_REF ${PULL_BASE_REF}"
+IMG=${IMG_REGISTRY}/btp-manager:$PULL_BASE_REF
+echo "Referring to image: ${IMG}"
 
-MODULE_VERSION=${PULL_BASE_REF} make module-build
+## prepare security scanning configuration for the module template
```
Comment (PK85) [inline scripts/release.sh:35]: where do we print sec config file content?

File: scripts/create_scan_config.sh (hunk 1 above)
Comment (jaroslaw-pieszka) [reply scripts/create_scan_config.sh:18]: I have not found anything about these but this [file](https://github.tools.sap/neighbors-team/security-scanners/blob/main/responsibilities/teams.yaml).

Comment (jaroslaw-pieszka) [reply scripts/create_scan_config.sh:18]: So there is assignment of the `btp-operator` to `gophers` but the default ID is wrong.

File: scripts/release.sh (hunk 2 above)
Comment (jaroslaw-pieszka) [reply scripts/release.sh:35]: cat <<EOF | tee ${FILENAME} in `create_scan_config.sh`

File: docs/contributor/03-10-release.md [hunk 3]
```diff
@@ -25,11 +25,10 @@ To create a release, follow these steps:
...
-6. `post-btp-manager-module-build` runs the `kyma alpha create module` command, which creates a Kyma module, and pushes the image to the registry. 
-Finally, the job uploads the `template.yaml`,`template_control_plane.yaml`, `btp-manager.yaml` and `btp-operator-default-cr.yaml` files to the btp-manager release as release assets.
-1. The GitHub action waits for the `template.yaml` asset in the GitHub release and for images in the Docker registry.
-2. The GitHub action fetches the module image and runs E2E tests on the k3s cluster with the specified credentials. 
-3. If unit tests and E2E tests are completed successfully, GitHub action publishes the release.
+6. `post-btp-manager-module-build` runs the `kyma alpha create module` command, which creates a Kyma module and pushes the image to the registry. Kyma CLI is called with `--sec-scanners-config` flag and uses dynamically created file to configure security scanning settings in the module template. Finally, the job uploads the `template.yaml`,`template_control_plane.yaml`, `btp-manager.yaml` and `btp-operator-default-cr.yaml` files to the btp-manager release as release assets.
```
Comment (IwonaLanger) [inline docs/contributor/03-10-release.md:28]:
```suggestion
6. `post-btp-manager-module-build` runs the `kyma alpha create module` command, which creates a Kyma module and pushes the image to the registry. Kyma CLI is called with the `--sec-scanners-config` flag and uses a dynamically created file to configure security scanning settings in the module template. Finally, the job uploads the `template.yaml`,`template_control_plane.yaml`, `btp-manager.yaml` and `btp-operator-default-cr.yaml` files to the btp-manager release as release assets.
```

File: docs/contributor/03-10-release.md [hunk 4]
```diff
@@ -25,11 +25,10 @@ To create a release, follow these steps:
...
+7. The GitHub action waits for the `template.yaml` asset in the GitHub release and for images in the Docker registry.
+8. The GitHub action fetches the module image and runs E2E tests on the k3s cluster with the specified credentials. 
+9. If unit tests and E2E tests are completed successfully, GitHub action publishes the release.
```
Comment (IwonaLanger) [inline docs/contributor/03-10-release.md:31]:
```suggestion
9. If the unit tests and E2E tests are completed successfully, the GitHub action publishes the release.
```

**Review by szwedm — APPROVED** [review]
/lgtm

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**